proc/process.c

```diff
@@ -1096,15 +1098,31 @@ static void process_exec(thread_t *current, process_spawn_t *spawn)
 #ifndef NOMMU
 	vm_mapCreate(&current->process->map, (void *)(VADDR_MIN + SIZE_PAGE), (void *)VADDR_USR_MAX);
 	proc_changeMap(current->process, &current->process->map, NULL, &current->process->map.pmap);
+	(void)i;
 #else
+	pmap_create(&current->process->map.pmap, NULL, NULL, NULL);
```

Comment:
Can `pmap_create()` fail on any arch? Or do we assume that armv7m's implementation always succeeds?

Reply:
Imho we can change the HAL API, so pmap_create has no return code - none of the architectures has any error path anyway